### DISCUSSION TITLE
How to change print scaling factor ?

### DISCUSSION THREAD

**Discussion by holicc** [qax-os/excelize#1289] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19146591/181044874-5c80a126-6344-413c-ae29-c46bf38b480b.png)

**Comment by xuri** [comment, marked as answer]
Sorry, this library doesn't support changing the print scaling. I will add this support later.

**Reply by holicc** [reply]
Do you have any ideas how to do solve this ? I'd like to make a PR, i'm really need this now . pls

**Reply by xuri** [reply]
Sorry for my previous reply, this library already supports changing the print scaling. Please specify the [`PageLayoutScale`](https://pkg.go.dev/github.com/xuri/excelize/v2#PageLayoutScale) field with the [`SetPageLayout`](https://pkg.go.dev/github.com/xuri/excelize/v2#File.SetPageLayout) function.

**Reply by holicc** [reply]
oooooh ! great ! thks  👍
